monitoring/dashboards/viya/sas-launched-jobs-node.json

```diff
@@ -3,7 +3,10 @@
     "list": [
       {
         "builtIn": 1,
-        "datasource": "-- Grafana --",
+        "datasource": {
+          "type": "datasource",
+          "uid": "grafana"
+        },
```

Comment:
Is this change in datasource meaningful?  Does it reflect a Grafana version difference or something else?  I see multiple instances of similar changes to datasource throughout this file.